azure-pipelines.yml

```diff
@@ -482,11 +491,11 @@ stages:
 
       - template: set-azp-variables-steps.yml@pipelines-yaml-templates
 
-      - task: Maven@3
+      - task: JFrogMaven@1
```

Comment:
@elrzn I am not sure we need the deployment connection for this... I will do a test on a separate PR.

But if it doesn't hurt, I suggest to merge it as is and make changes later.